apps/staking/app/mystakes/modules/BalanceModule.tsx

```diff
@@ -55,7 +55,10 @@ function useTotalStakedAmount(params?: { addressOverride?: Address }) {
     } else if (showMockNodes) {
       return [];
     }
-    return data?.stakes ?? [];
+    if (data && 'stakes' in data && Array.isArray(data.stakes)) {
+      return data.stakes;
```

Comment:
You could probably do something like you have down below if you just want 1 return

```tsx
const stakes = 'stakes' in data && Array.isArray(data.stakes) ? data.stakes : [];
```

Reply:
Id rather keep it like this, its cleaner

Reply:
Cool. As you want. Resolving